source/documentation/values/calculations.html.md.erb

```diff
@@ -258,3 +303,81 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
     padding-right: 20px;
   }
 <% end %>
+
+### [`round()`]
+<% impl_status dart: "1.64.0", libsass: false, ruby: false, feature: "Special function syntax" do %>
+  LibSass, Ruby Sass, and versions of Dart Sass prior to 1.64.0 *always* parse
+  `round()` a as Sass function. To create a plain CSS calculation function
+  call for those implementations, you can write something like
```

Comment:
"plain CSS calculation function call" is a little confusing here, since a "function call" and a "calculation" are usually different things in CSS. I'd probably just say "plain CSS calculation".